src/chainparams.cpp

```diff
@@ -283,12 +283,12 @@ class CMainParams : public CChainParams {
         chainTxData = ChainTxData{
             1577552136,
             8015431,
-            0.04004504697221486         
+            0.04004504697221486
         };
 
         // reorg
         consensus.nMaxReorgDepth = 5;
-        consensus.nMaxReorgDepthEnforcementBlock = 388000;
+        consensus.nMaxReorgDepthEnforcementBlock = 4760000;
```

Comment:
this is wayyy too high lol 
current block is 2549132, so maybe 2650000 is better or something like that.

Reply:
nevermind. lol